components/outputs/image.qmd

```diff
@@ -65,7 +65,6 @@ listing:
         <p>Set `ui.output_image(Fill=True)` to allow the image to grow or shrink to fill its container as the app is resized.</p>
         
         <p>You can use an image as an input widget, collecting the locations of user clicks, double clicks, hovers, and brushes. To do this, follow the instructions provided for [plots](TO DO: ADD LIN KTO THE DETAILS SECTION OF THE PLOTS PAGE).</p>
-
```

Comment:
I can't comment directly on the line because it wasn't changed in the file, but the shinylive link shows a datatable instead of an image.

Reply:
Thanks!